abi/src/main/java/org/web3j/abi/datatypes/generated/Int104.java

```diff
@@ -1,7 +1,6 @@
 package org.web3j.abi.datatypes.generated;
 
 import java.math.BigInteger;
-
```

Comment:
The newline was removed when the datatype classes were re-generated.

Reply:
It was probably IntelliJ that put the spaces in, as I've configured my imports to separate out web3j imports from third-party and core Java.

I don't think it's possible for Java Poet to organise the imports for you, but using this ordering if preferable. That being said, as its generated code it's not essential.